src/components/layouts/Footer.jsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from "react";
+
+
+function Footer(props) {
+  return (
+
+<footer className="footer p-10 bg-base-200 text-base-content">
+  <div>
+    <span className="footer-title">Services</span>
+    <a className="link link-hover">Branding</a>
+    <a className="link link-hover">Design</a>
+    <a className="link link-hover">Marketing</a>
+    <a className="link link-hover">Advertisement</a>
+  </div>
+  <div>
+    <span className="footer-title">Company</span>
+    <a className="link link-hover">About us</a>
+    <a className="link link-hover">Contact</a>
+    <a className="link link-hover">Jobs</a>
+    <a className="link link-hover">Press kit</a>
```

Comment:
ini nanti bisa diganti pake `NavLink` ya jadi kyk `<NavLink to="/about" className="link link-hover">About us</NavLink>`